website/src/data/users.js

```diff
@@ -476,6 +476,14 @@ const users = [
     source: 'https://github.com/linyuxuanlin/Wiki_Docusaurus',
     tags: ['large', 'personal'],
   },
+  {
+    title: 'PptxGenJS',
+    description: 'PowerPoint JavaScript Library',
+    preview: require('./showcase/pptxgenjs.png'),
+    website: 'https://gitbrent.github.io/pptxgenjs/',
```

Comment:
```suggestion
    website: 'https://gitbrent.github.io/PptxGenJS/',
```

looks the site url was not working because of case sensitivity